Gatsby Develop Unexpected token import error

Gatsby version: 1.8.8
node.js version: 8.1.2
OS version: 10.12.6

I created a starter with 
`$ gatsby new myblog`

I didn't modify any files. I just ran:
`$ npm run develop`

This was the output:
```
DRS:myblog shoemakerdr$ npm run develop

> gatsby-starter-default@1.0.0 develop /Users/shoemakerdr/dev/JS/myblog
> gatsby develop

success open and validate gatsby-config.js — 0.007 s
success copy gatsby files — 0.026 s
success source and transform nodes — 0.037 s
success building schema — 0.137 s
success createLayouts — 0.032 s
success createPages — 0.007 s
success createPagesStatefully — 0.025 s
success extract queries from components — 0.134 s
success run graphql queries — 0.021 s
success write out page data — 0.004 s
success update schema — 0.096 s

info bootstrap finished - 2.1 s

error There was an error compiling the html.js component for the development server.

See our docs page on debugging HTML builds for help https://goo.gl/yL9lND

  Error: Unexpected token import

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-default@1.0.0 develop: `gatsby develop`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gatsby-starter-default@1.0.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/shoemakerdr/.npm/_logs/2017-08-15T00_41_51_105Z-debug.log
DRS:myblog shoemakerdr$
```

--- a/packages/gatsby/src/utils/babel-config.js
+++ b/packages/gatsby/src/utils/babel-config.js
@@ -151,7 +151,7 @@ module.exports = async function babelConfig(program, stage) {
       {
         loose: true,
         uglify: true,
-        modules: false,
+        modules: `commonjs`,
         targets: {
           browsers: program.browserslist,
         },

